package.json

```diff
@@ -112,7 +112,7 @@
     "i18next-browser-languagedetector": "^6.1.4",
     "i18next-http-backend": "^1.4.1",
     "inobounce": "^0.1.2",
-    "interactjs": "1.4.0-alpha.17",
+    "interactjs": "1.10.27",
```

Comment:
Should there be a `^` on the version, or does the version of this need to be completely locked?